docs/draft-alakuijala-brotli-07.nroff

```diff
@@ -1434,52 +1434,52 @@ meta-block is the last one. The format of the meta-block header is
-                         otherwise MSKIPLEN is 0 (if MSKIPBYTES is
-                         greater than 1, and the last byte is all
-                         zeros, then the stream should be rejected
-                         as invalid)
-
-                 0 - 7 bits: fill bits until the next byte boundary,
-                             must be all zeros
-
-                 MSKIPLEN bytes of metadata, not part of the
-                         uncompressed data or the sliding window
-
-     MNIBBLES x 4 bits: MLEN - 1, where MLEN is the length
-              of the meta-block uncompressed data in bytes (if the
-              number of nibbles is greater than 4, and the last
-              nibble is all zeros, then the stream should be
-              rejected as invalid)
-
-      1 bit:  ISUNCOMPRESSED, if set to 1, any bits of compressed
-              data up to the next byte boundary are ignored, and
-              the rest of the meta-block contains MLEN bytes of
-              literal data; this field is only present if the
-              ISLAST bit is not set (if the ignored bits are not
-              all zeros, the stream should be rejected as invalid)
+   1 bit:  ISLAST, set to 1 if this is the last meta-block
+   1 bit:  ISLASTEMPTY, if set to 1, the meta-block is empty;
+           this field is only present if ISLAST bit is set -- if
+           it is 1, then the meta-block and the brotli stream ends
+           at that bit, with any remaining bits in the last byte
+           of the compressed stream filled with zeros (if the
+           fill bits are not zero, then the stream should be
+           rejected as invalid)
+   2 bits: MNIBBLES, # of nibbles to represent the uncompressed
+           length, encoded as follows: if set to 3, MNIBBLES is 0,
+           otherwise MNIBBLES is the value of this field plus 4.
+           If MNIBBLES is 0, the meta-block is empty, i.e. it does
+           not generate any uncompressed data. In this case, the
+           rest of the meta-block has the following format:
+
+              1 bit:  reserved, must be zero
+
+              2 bits: MSKIPBYTES, # of bytes to represent metadata
+                      length
+
+              MSKIPBYTES x 8 bits: MSKIPLEN - 1, where MSKIPLEN is
+                      the number of metadata bytes; this field is
+                      only present if MSKIPBYTES is positive,
+                      otherwise MSKIPLEN is 0 (if MSKIPBYTES is
+                      greater than 1, and the last byte is all
+                      zeros, then the stream should be rejected
+                      as invalid)
+
+              0 - 7 bits: fill bits until the next byte boundary,
+                          must be all zeros
+
+              MSKIPLEN bytes of metadata, not part of the
+                      uncompressed data or the sliding window
+
+   MNIBBLES x 4 bits: MLEN - 1, where MLEN is the length
+            of the meta-block uncompressed data in bytes (if the
```

Comment:
This should be indented one space less, to match the previous paragraph on the same level.